core/Updates/4.0.0-b1.php

```diff
@@ -48,11 +51,35 @@ public function getMigrations(Updater $updater)
             $migrations[] = $this->migration->plugin->activate('CustomJsTracker');
         }
 
+        if ($this->usesGeoIpLegacyLocationProvider()) {
+            // activate GeoIp2 plugin for users still using GeoIp2 Legacy (others might have it disabled on purpose)
+            $migrations[] = $this->migration->plugin->activate('GeoIp2');
+        }
+
+        // remove old options
+        $migrations[] = $this->migration->db->sql('DELETE FROM `' . Common::prefixTable('option') . '` WHERE option_name IN ("geoip.updater_period", "geoip.loc_db_url", "geoip.isp_db_url", "geoip.org_db_url")');
+
```

Comment:
those change are unrelated to this PR. Guess they are a result of a merge conflict maybe? 🤔

Reply:
Merged again